packages/storybook_flutter/CHANGELOG.md

```diff
@@ -1,96 +1,96 @@
 ## 0.9.0-dev.1
 
- - **FEAT**: Update plugins.
- - **FEAT**: Update plugins.
- - **DOCS**: Update example.
- - **CHORE**: Update pubignore.
+- **FEAT**: Update plugins.
```

Comment:
Please revert all the changes here, they are not needed.

Reply:
Oh, I see now. These changes are made by formatting the file, which VS Code does automatically. I'll manually revert them 👍🏻